gnocchi/tests/test_carbonara.py

```diff
@@ -84,19 +84,19 @@ def test_duplicate_timestamps(self):
              datetime64(2014, 1, 1, 12, 0, 9)],
             [10, 23])
         self.assertEqual(2, len(ts))
-        self.assertEqual(10.0, ts[0])
-        self.assertEqual(23.0, ts[1])
+        self.assertEqual(10.0, ts[0][1])
+        self.assertEqual(23.0, ts[1][1])
 
         ts.set_values([(datetime64(2014, 1, 1, 13, 0, 10), 3),
                        (datetime64(2014, 1, 1, 13, 0, 11), 9),
                        (datetime64(2014, 1, 1, 13, 0, 11), 8),
                        (datetime64(2014, 1, 1, 13, 0, 11), 7),
                        (datetime64(2014, 1, 1, 13, 0, 11), 4)])
         self.assertEqual(4, len(ts))
-        self.assertEqual(10.0, ts[0])
-        self.assertEqual(23.0, ts[1])
-        self.assertEqual(3.0, ts[2])
-        self.assertEqual(4.0, ts[3])
+        self.assertEqual(10.0, ts[0][1])
+        self.assertEqual(23.0, ts[1][1])
+        self.assertEqual(3.0, ts[2][1])
+        self.assertEqual(9.0, ts[3][1])
```

Comment:
weird, github hides what i typed for object name. measures_metricid_uuid_timestamp. done.